src/sentry/deletions/defaults/group.py

```diff
@@ -84,9 +90,9 @@ def chunk(self):
             limit=self.DEFAULT_CHUNK_SIZE,
             referrer="deletions.group",
             orderby=["-timestamp", "-event_id"],
-            tenant_ids={"organization_id": self.groups[0].project.organization_id}
-            if self.groups
-            else None,
+            tenant_ids=(
+                {"organization_id": self.groups[0].project.organization_id} if self.groups else None
```

Comment:
This is now failing with: `error: Value of type "Iterable[Group]" is not indexable`